samples/apps/anomaly-detection-app/requirements.txt

```diff
@@ -1,6 +1,5 @@
 grpcio==1.53.0
 grpcio-tools==1.53.0
 Flask==2.2.5
-numpy==1.21.4
-protobuf==4.21.6
-scikit_learn==1.0.1
```

Comment:
scikit_learn is still a dependency for the sample app

Reply:
The scikit_learn is pulled in by the Dockerfile

Reply:
add scikit_learn back but use default version.